build.proj

```diff
@@ -10,6 +10,28 @@
 
   <Import Project="dir.traversal.targets" />
 
+  <PropertyGroup>
+    <TraversalBuildDependsOn>
+      BatchRestorePackages;
+      $(TraversalBuildDependsOn);
+    </TraversalBuildDependsOn>
+  </PropertyGroup>
+
+  <Target Name="BatchRestorePackages">
+    <Message Importance="High" Text="Restoring all packages..." />
+    <!-- restore all project.jsons in one pass for perf & to avoid concurrency problems with dnu -->
+    <!-- include ToolsDir to restore test-runtime\project.json as well -->
+    <Exec Command="$(DnuRestoreCommand) &quot;$(MSBuildProjectDirectory)\src&quot; &quot;$(ToolsDir)&quot;" StandardOutputImportance="Low" CustomErrorRegularExpression="^Unable to locate .*" />
```

Comment:
There isn't any extra options needed to tell it to recursively look from the files?